src/accordion/accordion-group.component.ts

```diff
@@ -76,9 +76,14 @@ export class AccordionPanelComponent implements OnInit, OnDestroy {
     this.accordion.removeGroup(this);
   }
 
-  toggleOpen(): void {
+  toggleOpen(event: Event): void {
```

Comment:
This is public API, adding mandatory option is potentially a breaking change. Please make it optional